protos/search3.proto

```diff
@@ -1,21 +1,32 @@
 // -*-java-*-
```

Comment:
this file again isn't needed because we won't be generating code from this branch